code/core/system/proxy-velodyne16/include/velodyne16Decoder.h

```diff
@@ -16,73 +16,73 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
- 
+
 #ifndef VELODYNE16DECODER_H_
 #define VELODYNE16DECODER_H_
 
 #include <memory>
 
-#include <opendavinci/odcore/io/StringListener.h>
+#include "opendavinci/generated/odcore/data/SharedPointCloud.h"
 #include "opendavinci/odcore/data/Container.h"
 #include "opendavinci/odcore/wrapper/SharedMemory.h"
-#include "opendavinci/generated/odcore/data/SharedPointCloud.h"
+#include <opendavinci/odcore/io/StringListener.h>
 
 namespace opendlv {
 namespace core {
 namespace system {
 namespace proxy {
-    using namespace odcore::wrapper;
+using namespace odcore::wrapper;
 
-    // This class will handle bytes received via a UDP socket.
-    class velodyne16Decoder : public odcore::io::StringListener {
-    private:
-        /**
+// This class will handle bytes received via a UDP socket.
+class velodyne16Decoder : public odcore::io::StringListener {
```

Comment:
In OpenDaVINCI and OpenDLV.core, we use CamelCase for class names and always start with a capital letter. Velodyne16Decoder would be the expected class name in this case.